netam/dasm.py

```diff
@@ -139,7 +139,8 @@ def prediction_pair_of_batch(self, batch):
             raise ValueError(
                 f"log_neutral_aa_probs has non-finite values at relevant positions: {log_neutral_aa_probs[mask]}"
             )
-        log_selection_factors = self.model(aa_parents_idxs, mask)
+        keep_token_mask = mask | sequences.token_mask_of_aa_idxs(aa_parents_idxs)
```

Comment:
Would you mind adding a comment just above here explaining that we want to keep the special tokens in the AA string so that the model can see them?